src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -709,6 +710,15 @@ public class SimpleEvents {
 					.requiredPlugins("Paper 1.16+");
 		}
 
+			Skript.registerEvent("Player Pickup Arrow", SimpleEvent.class, PlayerPickupArrowEvent.class, "[player] (pick[ing| ]up [an] arrow|arrow pick[ing| ]up)")
```

Comment:
```suggestion
			Skript.registerEvent("Player Pickup Arrow", SimpleEvent.class, PlayerPickupArrowEvent.class, "[player] (pick[ing| ]up [of an] arrow|arrow pick[ing| ]up)")
```

Reply:
Can we make the keyword `of` optional?

Reply:
IMO it should be optional as with and without is correct

Reply:
On pickup an arrow is not proper

Reply:
Thats fair, but `on picking up an arrow` is correct too

Reply:
> On pickup an arrow is not proper

That's fine, we can't create syntaxes for all patterns